src/main/java/com/epam/training/designpatterns/fastfoodrestaurant/tables/Client.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.training.designpatterns.fastfoodrestaurant.tables;
+
+import com.epam.training.designpatterns.fastfoodrestaurant.foods.Food;
+import com.epam.training.designpatterns.fastfoodrestaurant.foods.Menu;
+import com.epam.training.designpatterns.fastfoodrestaurant.staff.Server;
+
+import java.util.Random;
+import java.util.concurrent.Semaphore;
+
+/**
+ * Knows the server, and the menu
+ */
+public class Client implements Runnable{
```

Comment:
Légyszi rendezd a metódusokat a következőképpen:
- constructor elöl
- public metódusok
- private metódusok
- az egyes private metódusok sorrendje az legyen, amilyen sorrendben hívják őket a public metódusokból vagy másik private metódusokból

A többi osztályyban is csináld ezt meg! Köszi!